glusterd2/commands/global/getglobaloptions.go

```diff
@@ -0,0 +1,46 @@
+package globalcommands
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/gluster/glusterd2/glusterd2/cluster"
+	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
+	"github.com/gluster/glusterd2/glusterd2/store"
+	"github.com/gluster/glusterd2/pkg/api"
+	"github.com/gluster/glusterd2/pkg/errors"
+	"github.com/gorilla/mux"
+)
+
+func getGlobalOptionsHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	optname := mux.Vars(r)["optname"]
+
+	var c cluster.Cluster
+	resp, err := store.Store.Get(context.TODO(), cluster.ClusterPrefix)
```

Comment:
Create this as function in separate file `store-utils.go`. So that all store related functions will be in one place.

Reply:
Sure.